core/lib/dal/migrations/20231208134254_drop_old_witness_generation_related_tables.down.sql

```diff
@@ -0,0 +1,83 @@
+CREATE TABLE IF NOT EXISTS witness_inputs
+(
+    l1_batch_number BIGINT NOT NULL PRIMARY KEY,
+    merkle_tree_paths BYTEA,
+    created_at TIMESTAMP NOT NULL,
+    updated_at TIMESTAMP NOT NULL,
+    status TEXT NOT NULL,
+    time_taken TIME DEFAULT '00:00:00'::TIME WITHOUT TIME ZONE NOT NULL,
+    processing_started_at TIMESTAMP,
+    error VARCHAR,
+    attempts INTEGER DEFAULT 0 NOT NULL,
+    merkel_tree_paths_blob_url TEXT,
+    is_blob_cleaned boolean DEFAULT false NOT NULL,
+    protocol_version INTEGER
+    CONSTRAINT witness_inputs_prover_protocol_version_fkey REFERENCES prover_protocol_versions
+);
+CREATE INDEX IF NOT EXISTS witness_inputs_blob_cleanup_status_index ON witness_inputs (status, is_blob_cleaned);
```

Comment:
not sure, but I think we'll never be able to roll back such migrations. We could even consider leaving it empty or adding some exception so that it's clear that we cannot roll back. Even if we make this migration work, I think most of the others are broken/not compatible